spec/order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'order'
+require 'menu'
+
+describe Order do
+  let(:menu) { double :menu }
+  subject(:order) { described_class.new(menu) }
+  it "can calculate the total cost of the order" do
+    expect(order).to respond_to(:total)
+  end
+
+  it "records the quantity of each dish ordered" do
+    expect(order).to respond_to(:total_quantity)
+  end
+
+  it "returns a list of dishes in the order" do
+    expect(order).to respond_to(:list_dishes)
+  end
+
+  describe "#total" do
+    let(:dish) { double :dish, price: 5 }
+    let(:menu) { double :menu }
+    it "returns an order total of 10 for two 5 pound dishes ordered" do
+      test_order = Order.new
+      test_order.add(dish, 2)
+      expect(test_order.total).to eq 10
+    end
+  end
+
+  describe "#add" do
+    it "adds dishes to the order" do
+      expect(order).to respond_to :add
+    end
+  end
+
```

Comment:
maybe you could add a test to check if the correct dishes have been added to the order?